src/video_emotion/api_helper.py

```diff
@@ -1,11 +1,30 @@
-import json
-import os
+from typing import Dict
 
 from video_emotion.extract_classifier import classify_video
 from database.video_emotion_no_song import VideoEmotionNS
 from database.video_emotion import VideoEmotion
 
-def process_data_and_extract_emotions(video_id, video_path, time_range):
+
+def process_data_and_extract_emotions(video_id: int, video_path: str, time_range:Dict[str,int]) -> bool:
+    """Clasifies the emotions in a video
+
+    Parameters
+    ----------
+    video_id
```

Comment:
The parameters should also show the type. Like this:
```
    Parameters
    ----------
    video_id : int
```

Also, make sure of what the type of the video_id parameter is. I think it is string.